dietpi-install.sh

```diff
@@ -38,16 +25,35 @@ xz -d "$IMAGE_NAME"
 IMAGE_NAME=${IMAGE_NAME%.xz}
 sleep 3
 
-# import the qcow2 file to the default virtual machine storage
+# Import the qcow2 file to the specified storage
+echo "Importing disk image to storage..."
 qm importdisk "$ID" "$IMAGE_NAME" "$STORAGE"
 
-# Set vm settings
+# Check if the disk was imported correctly
+if ! qm config "$ID" | grep -q "unused0"; then
+    echo "Error: Failed to import disk for VM $ID"
+    exit 1
+fi
+
+# Retrieve the disk path and print for user
+DISK_PATH=$(qm config "$ID" | grep "unused0" | awk '{print $2}')
```

Comment:
`awk` can do both 🙂:
```suggestion
DISK_PATH=$(qm config "$ID" | awk '/unused0/{print $2}')
```

Reply:
Perfect 👍